src/bundle/Resources/public/scss/_alloyeditor-link-edit.scss

```diff
@@ -0,0 +1,96 @@
+.ez-ae-edit-link {
+    display: flex;
+    flex-direction: column;
+    padding: .5rem;
+
+    .ez-ae-edit-link__row {
+        display: flex;
+        align-items: flex-end;
+
+        &:not(:last-child) {
+            margin-bottom: 1rem;
+        }
+
+        &.ez-ae-edit-link__row--actions {
+            justify-content: flex-end;
+        }
+    }
+
+    .ez-ae-edit-link__block {
```

Comment:
Please, use sass here. There's no need to duplicate `ez-ae-edit-link` here and there. Please, use `&` instead.